Oracle.md

```diff
@@ -198,6 +200,27 @@ The TLV serialization of oracle announcements is as follow.
 
 where `signature` is a Schnorr signature over a sha256 hash of the serialized `oracle_event`, using the tag `announcement/v0`.
 
+## Oracle Attestations
+
+After an event occurs, and the oracle creates signatures to attest the outcome, it needs to give them to users.
+An oracle can use an attestation tlv to give users this information.
+
+The TLV serialization of oracle attestations is as follows.
+
+#### Version 0 `oracle_attestation`
+
+1. type: 55400 (`oracle_attestation_v0`)
+2. data:
+    * [`x_point`:`oracle_public_key`]
```

Comment:
Personally I have something similar, but with two more things:
* the event id
* the list of values that were signed
I guess we can argue if these are necessary or not, but at least on my side it makes things much easier to handle.

Reply:
These seem reasonable to me.